dashboard/app/controllers/courses_controller.rb

```diff
@@ -98,7 +98,7 @@ def edit
     raise ActiveRecord::ReadOnlyRecord if @unit_group.try(:plc_course)
     @unit_group_data = {
       course_summary: @unit_group.summarize(@current_user, for_edit: true),
-      script_names: Unit.all.map(&:name),
+      script_names: Unit.all.select {|unit| unit.is_course? == false}.map(&:name),
```

Comment:
This is the main change, it removes all units that are standalone courses from showing up in the dropdown.

Reply:
I'm a little confused why this method exists alongside the `is_course` serialized property and which one is the source of truth, but for this kind of change I think this is good enough